astropy/cosmology/tests/test_cosmology.py

```diff
@@ -1623,3 +1623,12 @@ def test_z_at_value_roundtrip():
         fval = func(z)
         assert allclose(z, funcs.z_at_value(func, fval, zmax=1.5),
                         rtol=2e-8)
+
```

Comment:
Insert the following decorator in front of the new test function.

@pytest.mark.skipif('not HAS_SCIPY')

Reply:
Thanks, done.